src/nvim/mouse.h

```diff
@@ -20,6 +20,7 @@ enum {
   MOUSE_FOLD_CLOSE = 0x200,  ///< clicked on '-' in fold column
   MOUSE_FOLD_OPEN  = 0x400,  ///< clicked on '+' in fold column
   MOUSE_WINBAR     = 0x800,  ///< in window toolbar
+	MOUSE_NUMCOL     = 0x1000, ///< in 'numbercolumn'
```

Comment:
```suggestion
  MOUSE_NUMCOL     = 0x1000, ///< in 'numbercolumn'
```

Reply:
Should I only follow the lint suggestion or shift the whole comment column?